spec/requests/resource_operations_spec.rb

```diff
@@ -70,7 +70,7 @@
   end
 
   describe 'POST /articles' do
-    subject(:last_response) { post("/articles", '{ "data": { "type": "articles" } }') }
+    subject(:last_response) { post("/articles", '{ "data": { "id": "1", "type": "articles" } }') }
```

Comment:
The reason is as we're forcing the presence of `external_id` in the db we need to set it from somewhere. In a production env it could be set from a DB pgsql function or something (UUID). 

> Auto-increment-like behavior no longer works, I assume.

You're correct, but I don't think it's a big issue for the specs right?